python/mxnet/ndarray.py

```diff
@@ -1197,33 +1199,51 @@ def _ufunc_helper(lhs, rhs, fn_array, fn_scalar, lfn_scalar, rfn_scalar=None):
 #pylint: enable= too-many-arguments, no-member, protected-access
 
 def add(lhs, rhs):
-    """Add arguments, element-wise with broadcasting.
+    """Returns element-wise sum of the input arrays with broadcasting.
 
-    Equivalent to ``lhs + rhs``
+    Equivalent to ``lhs + rhs`` and ``mx.nd.broadcast_add(lhs, rhs)``.
+
+    .. note::
+
+        If the corresponding dimensions of two arrays have the same size or one of them has size 1
+    then, the arrays are considered as broadcastable to a common shape.
```

Comment:
"as" is unnecessary after considered. However, I'm not a native speak. @zackchase ?